pages/about.html

```diff
@@ -5,4 +5,20 @@
 home: false
 ---
 
-<h2>About Content Goes Here</h2>
\ No newline at end of file
+<div class='content-container'>
+    <div class='content-header'>
+        <h2>What is Civic Tech?</h2>
+        <p>Civic tech bridges the gap between the civic space and technology. It’s utilizing technology practices like
+            design, engineering, data science, etc. to optimize the processes for civic organizations.</p>
+    </div>
+    <div class='quote-content'>
+        <div class='quote-paragraph-container'>
+            <p class='quote-paragraph'>“...those of us who work in civic tech want public digital goods to be as good as
+                the ones made by
+                commercial
+                entities like Apple or Google—and we want public digital infrastructure to be as good, too.” </p>
```

Comment:
Small thing--Can you clean up the odd spacing in this quote? Lots of line breaks in the code